sdk/azidentity/managed_identity_client.go

```diff
@@ -28,12 +28,14 @@ import (
 
 const (
 	arcIMDSEndpoint          = "IMDS_ENDPOINT"
+	defaultIdentityClientID  = "DEFAULT_IDENTITY_CLIENT_ID"
```

Comment:
Could get better grouping if we named it starting with IDENTITY
```suggestion
	identityDefaultClientID  = "IDENTITY_DEFAULT_CLIENT_ID"
```

Reply:
This is Azure ML's variable--we have to use the name they chose

Reply:
Can we push back on them to change that?

Reply:
I suppose we could try but because they've already shipped and documented this variable we'd be asking them to ship and document a new version of their managed identity endpoint. I imagine it would be very difficult to convince them changing this name is worth that effort.